static/styles/portico.css

```diff
@@ -1272,7 +1272,7 @@ input.new-organization-button {
 }
 
 .markdown h3 {
-    margin-top: 30px;
+    margin-top: 20px;
```

Comment:
I tried just removing this item altogether, and it looked fine; maybe we don't need it?  We don't have something similar for h2...